src/net.cpp

```diff
@@ -81,6 +81,9 @@ struct ListenSocket
 };
 }
 
+/** Services this node implementation cares about */
+static const uint64_t nRelevantServices = NODE_NETWORK;
```

Comment:
again here, are we not filtering outbound connections based on NODE_NETWORK?  If we connect here and don't get node network (a pruned node) is the connection not going to get dropped?